### PR TITLE
Fix PHP 8 error: `Function libxml_disable_entity_loader() is deprecated`

### PR DESCRIPTION
**Pull request by Slamdunk**

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ jobs:
         uses: shivammathur/setup-php@v2 #https://github.com/shivammathur/setup-php
         with:
           php-version: ${{ matrix.php-versions }}
+          ini-values: error_reporting=E_ALL
```
Comment (Slamdunk) [inline .github/workflows/php.yml:24]: See https://github.com/shivammathur/setup-php/issues/450

**Comment by tvdijen** [comment]
This only bypasses the deprecated method on PHP 8+, but that's wrong.. We need to know what alternative replaces this call.

**Comment by Slamdunk** [comment]
> We need to know what alternative replaces this call.

Nothing: https://www.php.net/manual/en/migration80.deprecated.php#migration80.deprecated.libxml

Same fix was applied in `laminas-xml` package: https://github.com/laminas/laminas-xml/pull/3

**Comment by Slamdunk** [comment]
> I'm surprised to see a PR in this (yet) unused repository :)

I need to upgrade my production to PHP 8, and v4 on `saml2` doesn't support it, so I'm on `dev-master` across all your libraries

**Comment by tvdijen** [comment]
Ah, well in this case I must warn you that saml2 `master` is really bleeding edge and work in progress!
Good to see we have a voluntary beta-tester though :rofl:

**Comment by tvdijen** [comment]
I see no harm in adding php 8 to the supported versions for saml2 v4, so if that would help you, please let me know

**Comment by Slamdunk** [comment]
> Ah, well in this case I must warn you that saml2 `master` is really bleeding edge and work in progress!

Easy to tame on projects with 100% Mutation testing score + max level on static analysis tools

**Comment by Slamdunk** [comment]
> I see no harm in adding php 8 to the supported versions for saml2 v4, so if that would help you, please let me know

As a OSS maintainer myself, I would suggest you to support every actively supported PHP version until you release a new stable of your own.
In other words, `saml2:v4` should indeed support PHP 8, since `v5` isn't ready yet.

**Pull request merged**